### PR TITLE
Fixed setting of useTime in useTime() method

### PR DESCRIPTION
**Pull request by vysinsky**

### CODE REVIEW

**Comment by vojtech-dobes** [comment]
Hm, this is weird. Yeah, it's brutal bug, but this is not solution I think. The method doesn't support deactivation of time.

**Comment by vysinsky** [comment]
I think it does now

**Comment by vojtech-dobes** [comment]
I think it still doesn't - you don't cancel previously defined rule.

File: src/CalendarPicker.php [hunk 1]
```diff
@@ -207,6 +209,10 @@ public function getValue()
 
 	public function getControl()
 	{
+		if ($this->useTime) {
```
Comment (vojtech-dobes) [inline src/CalendarPicker.php]: I think this is too late. When you process data after submit, `getControl` doesn't have to be called.

**Comment by vojtech-dobes** [comment]
I think easiest solution is to make `useTime` able to enable only.

File: src/CalendarPicker.php (hunk 1 above)
Comment (vysinsky) [inline src/CalendarPicker.php]: Ok. What do you suggest? Call this in `attached`?

**Comment by vysinsky** [comment]
> I think easiest solution is to make useTime able to enable only.

ok

**Comment by vojtech-dobes** [comment]
:+1:

**Pull request merged**